package/Dockerfile

```diff
@@ -35,4 +35,6 @@ RUN chmod +x /usr/bin/support-bundle-kit
 ADD hack/collector-* /usr/bin/
 RUN chmod +x /usr/bin/collector-*
 
+ADD hack/common /tmp/
```

Comment:
Because `common` only defines the variable or function, we should not execute this directly.
So I would like to put it on `/tmp/` instead `/usr/bin/`.